app/views/users/edit.html.erb

```diff
@@ -22,18 +22,16 @@
 <header class="crayons-layout crayons-layout--limited-l crayons-layout--1-col p-4">
   <% if @tab == "organization" && @organizations.present? %>
     <h1 class="crayons-title">Settings for
-      <span class="organization-dropdown">
-        <select id="organization-dropdown-select" onchange="window.location.href=this.value">
-          <% options = @organizations.pluck(:slug, :id).each do |arr| %>
-            <% arr[0] = "@#{arr[0]}" %>
-            <% arr[1] = "/settings/organization/#{arr[1]}" %>
-          <% end.to_h.merge("New Organization" => "/settings/organization/new") %>
-          <%= options_for_select(options, "/settings/organization/#{params[:org_id]}") %>
-          <%# For example: %>
-          <%# <option value="/settings/organization/1>@devteam</option> %>
-          <%# <option value="/settings/organization/new" selected>New Organization</option> %>
-        </select>
-      </span>
+      <select class="crayons-select lh-tight w-25 align-text-top" onchange="window.location.href=this.value">
```

Comment:
TLDR; I removed a `<span>` that wrapped the `<select>` and added new utility classes.